src/store/traction/ont/runs/actions.js

```diff
@@ -0,0 +1,142 @@
+import { handleResponse } from '@/api/ResponseHelper'
+
+const newRun = ({ commit }) => {
+  let run = {
+    id: 'new',
+    instrument_name: null,
+    state: null,
+    flowcell_attributes: [],
+  }
+
+  commit('setCurrentRun', run)
+}
+
+const createRun = async ({ getters, rootGetters }) => {
+  let run = getters.currentRun
+  let request = getters.runRequest
+
+  let instrument_id = getters.instruments.find((i) => i.name == run.instrument_name).id
+
+  let existingPools = rootGetters['traction/ont/pools']
+
+  let flowcell_attributes = run.flowcell_attributes
+    .filter((fc) => fc.flowcell_id && fc.tube_barcode)
+    .map((fc) => {
+      let pool = existingPools.find((p) => p.barcode == fc.tube_barcode)
+      let pool_id = pool ? pool.id : ''
+      return { ...fc, ...{ ont_pool_id: pool_id } }
```

Comment:
does the backend expect a pool_id? It would make sense for the backend to just require the pool barcode?

Reply:
Options:
- Fetch pool by barcode, and get id like that (rather than getting all the pools)
- Change back end
- Leave as is